code/game/machinery/_machinery.dm

```diff
@@ -115,7 +115,13 @@ Class Procs:
 	var/atom/movable/occupant = null
 	var/speed_process = FALSE // Process as fast as possible?
 	var/obj/item/circuitboard/circuit // Circuit to be created and inserted when the machinery is created
+	var/obj/item/card/id/prisoner/inserted_prisoner_id
```

Comment:
don't have these all in the top like this, not what I meant, it's cleaner to have them only in needed children